schema/channels.graphql

```diff
@@ -60,7 +60,10 @@ type Channel @entity {
   totalVideosCreated: Int!
 
   "Cumulative rewards claimed by this channel"
-  cumulativeRewardClaimed: BigInt
+  cumulativeRewardClaimed: BigInt!
+
+  "Cumulative rewards paid to this channel"
+  cumulativeRewardPaid: BigInt!
```

Comment:
I think it is hard now to know the difference between reward claimed vs paid.
```suggestion
  cumulativeReward: BigInt!
```
From what I understand we want to track `rewardClaimed + membershipMetadataPayment` 
So I think this is a more appropriate

Reply:
Alternatively you could just keep an extra `cumulativeRewardFromMembers` and then on atlas you could rank by `cumulativeRewardClaimed + cumulativeRewardFromMembers`